assets/schema_input.json

```diff
@@ -10,27 +10,39 @@
             "sample": {
                 "type": "string",
                 "pattern": "^\\S+$",
-                "errorMessage": "Sample name must be provided and cannot contain spaces"
+                "errorMessage": "Sample ID must be provided and cannot contain spaces",
+                "meta": ["sample"]
```

Comment:
If you use `"meta": ["id"]` here you won't need to modify the channel [afterwards](https://github.com/qbic-pipelines/createpanelofnormals/pull/1/files#diff-2dfce64501c68086d38cac9e05a1c1b683efacaf9bff552fabcb8f1ae309e05aR59)

Reply:
ah brilliant :)